src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -150,13 +151,18 @@
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.String.cs" />
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.UnsignedNumber.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsNETStandard)' == 'true'">
+  <ItemGroup Condition="'$(TargetsNETStandard)' == 'true' OR '$(TargetsNetFx)' == 'true'">
     <!-- Common or Common-branched source files -->
     <Compile Include="$(CommonPath)\System\Buffers\ArrayBufferWriter.cs">
       <Link>Common\System\Buffers\ArrayBufferWriter.cs</Link>
     </Compile>
+    <Reference Include="mscorlib" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.ValueTuple" />
+    <Reference Include="System.Numerics.Vectors" />
```

Comment:
Why did we end up needing these additional references? Should they be in a separate condition block where only `TargetsNetFx` is true?

Reply:
We can separate them out if you prefer, but this itemgroup was already applying for netstandard build and netfx. When in netstandard, these extra references will not do any harm as they are only facades in netstandard and don't have any typedefs, so the resulting assembly will be exactly the same. I wanted to keep the netstandard and netfx configurations as joint as possible when adding the extra configuration, which is why I didn't bother to add the extra itemgroup. That said, we can change this if you prefer.